app/components/primer/beta/button_group.rb

```diff
@@ -8,45 +8,45 @@ class ButtonGroup < Primer::Component
 
       # Required list of buttons to be rendered.
       #
-      # @param kwargs [Hash] The same arguments as <%= link_to_component(Primer::ButtonComponent) %> except for `size` and `group_item`.
+      # @param kwargs [Hash] The same arguments as <%= link_to_component(Primer::Beta::Button) %>
       renders_many :buttons, lambda { |**kwargs|
-        kwargs[:group_item] = true
-        kwargs[:size] = @size
+                               kwargs[:size] = @size
```

Comment:
What's going on with the indentation of this lambda?

Reply:
fixed